CHANGELOG.md

```diff
@@ -2,11 +2,74 @@
+* Added a different scope naming format for APIs that deal with explicit subscriptions by @shilpa-padgaonkar in https://github.com/camaraproject/Commonalities/pull/177
+* 'info' object, 'servers' added in chapter 11 of API Design Guidelines:  by @rartych in https://github.com/camaraproject/Commonalities/pull/214
+* Guidelines on device identification in Annex A of API Design Guidelines and device object usage in CAMARA_common.yaml by @jpengar in https://github.com/camaraproject/Commonalities/pull/233
+* `minItems: 1` & `maxItems: 1` for subscription `types` in event-subscription-template.yaml by @maxl2287 in https://github.com/camaraproject/Commonalities/pull/236
+* `SUBSCRIPTION_DELETED` as new `terminationReason` for CloudEvents by @maxl2287 in https://github.com/camaraproject/Commonalities/pull/238
+
+
+### Changed
+* API Design Guidelines updated with subscriptionMaxEvents by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/131
+* 'specversion' in CloudEvents as enum by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/187
+* 'datacontenttype' in CloudEvents as enum by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/193
+* API Design Guidelines updated on x-correlator format by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/194
+* API Design Guidelines for Notification Subscription by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/198
+* API Design Guidelines updated on scope definition including wildcard scopes by @shilpa-padgaonkar in https://github.com/camaraproject/Commonalities/pull/221
+* API Design Guidelines adapted to ICM Security and Interoperability Profile by @AxelNennker in https://github.com/camaraproject/Commonalities/pull/208
+* Error response model updated in chapter 6 of API Design Guidelines by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/213
+* Enhanced API-Testing-Guidelines.md by @jlurien in https://github.com/camaraproject/Commonalities/pull/203
+* Updated API versioning guidelines chapter 5 of API Design Guidelines by @tanjadegroot in https://github.com/camaraproject/Commonalities/pull/215
+  
+
+### Fixed
+* API Design Guidelines updated with character set guidance by @trehman-gsma in https://github.com/camaraproject/Commonalities/pull/143
+* Mandated '+' in all phoneNumber formats by @fernandopradocabrillo in https://github.com/camaraproject/Commonalities/pull/148
+* Linting rules problem with Traffic Influence API #161 by @VijayKesharwani in https://github.com/camaraproject/Commonalities/pull/169
+* CAMARA_common.yaml - bugs and typos fix by @fernandopradocabrillo in https://github.com/camaraproject/Commonalities/pull/174
+* API Design Guidelines updated on HTTPs usage by @AxelNennker in https://github.com/camaraproject/Commonalities/pull/205
+* Artifacts aligned with API Design Guidelines for Notification Subscription by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/219
```

Comment:
Suggestion to include as Fixed PRs:
https://github.com/camaraproject/Commonalities/pull/229
https://github.com/camaraproject/Commonalities/pull/234
https://github.com/camaraproject/Commonalities/pull/241

Reply:
ok, that is fine. Understood the point of "technical change".